content/guides/userguides/quick-start-guide.md

```diff
@@ -63,7 +63,7 @@ This metadata, also called a "Data Manifest" is in TSV file format and lists all
 
 To download the metadata from the **Export Selected Data** page:
 
-1. Go to **Download a File Manifest with Metadata for the Selected Data** and select **Start**
+1. Under **Download a File Manifest with Metadata for the Selected Data** and select **Request File Manifest**
```

Comment:
```suggestion
1. Under **Download a File Manifest with Metadata for the Selected Data**, select **Request File Manifest**

Reply:
@kayleemathews this does not seem to make sense the way it's written currently:

 "1. Under **Download a File Manifest with Metadata for the Selected Data** and select **Request File Manifest**